1-Weekly-Mission/js/signin.js

```diff
@@ -41,25 +42,37 @@ const setPasswordErrorMessage = (e) => {
   }
 };
 
-// form 전송
-const submitForm = (e) => {
+const submitForm = async (e) => {
   e.preventDefault();
-  // 이메일: test@codeit.com, 비밀번호: codeit101 으로 로그인 시도할 경우
-  if (
-    $emailBox.value === "test@codeit.com" &&
-    $passwordBox.value === "codeit101"
-  ) {
-    location.href = "./folder.html";
-  } else {
-    $emailErrorMsg.textContent = "이메일을 확인해주세요.";
-    $emailBox.classList.add("error_border");
-    $passwordErrorMsg.textContent = "비밀번호를 확인해주세요.";
-    $passwordBox.classList.add("error_border");
+
+  try {
+    const clientAccount = {
+      email: $emailBox.value,
+      password: $passwordBox.value,
+    };
+
+    const response = await fetchClient("sign-in", "POST", clientAccount);
+    const { data } = await response.json();
+
+    // 로그인 성공
+
+    if (response.status === 200) {
+      window.localStorage.setItem("accessToken", data.accessToken);
+    }
+  } catch (error) {
```

Comment:
```suggestion
  } catch {
```

error 가 따로 사용되지 않아서 `(error)`를 빼줘도 괜찮을 것 같습니다.